src/lib.rs

```diff
@@ -0,0 +1,44 @@
+#[macro_use]
```

Comment:
We probably need `#![cfg_attr(feature = "compat"), allow(unstable_name_collisions)]`